modules/eventing/pages/eventing-advanced-keyspace-accessors.adoc

```diff
@@ -698,286 +394,313 @@ Some example return values:
+    var result = couchbase.increment(src_col, ctr_meta);
+    if (result.success) {
+        log('success adv. increment: result', result);
+    } else {
+        log('failure adv. increment: id', ctr_meta.id, 'result', result);
+    }
+}
+----
+.Results
 [source,javascript]
 ----
-{"count": 23} // 23 is the current counter value
+{
+    "doc": {
+        "count": 24
+    },
+    "meta": {
+        "id": "key::1",
+        "cas": "1609374571840471040"
+    },
+    "success": true
+}
 ----
+====
+
+[#advanced-decrement-op]
+== Advanced DECREMENT Operation
 
-The _increment_ operation returns the post-increment value.
+`result = couchbase.decrement(binding, meta)`
 
-If the specified counter document does not exist, one is created with _count_ value as 0 and the structure noted above. And so, the first returned value will be 1.
+The DECREMENT operation lets you decrement the `count` field in a specific document.
 
-Due to limitations in KV engine API, this operation cannot currently manipulate full document counters.
+The document must have the structure `{ count: 23 }`, where 23 is the example counter value.
 
-There is no analogous _Basic Bucket Accessor_ operation to the _Advanced INCREMENT operation_.
+If the specified counter document does not exist, the operation creates a new document with a `count` value of 0. If the `count` value is 0, the first returned value is -1.
 
-Below is an example of the _Advanced INCREMENT operation_.
+The DECREMENT operation cannot manipulate full document counters because of limitations in the KV engine API.
 
+==== Example
+====
+.Operation
 [source,javascript]
 ----
 function OnUpdate(doc, meta) {
     log('input meta', meta);
-    log('input doc ', doc);
+    log('input doc', doc);
 
-    // if doc.count doesn't exist it will be created
+    // The operation creates a doc.count if it does not already exist
     var ctr_meta = {"id": "my_atomic_counter:1" };
-    var result = couchbase.increment(src_col,ctr_meta);
+    var result = couchbase.decrement(src_col, ctr_meta);
     if (result.success) {
-        log('success adv. increment: result',result);
+        log('success adv. decrement: result', result);
     } else {
-        log('failure adv. increment: id',ctr_meta.id,'result',result);
+        log('failure adv. decrement: id', ctr_meta.id, 'result', result);
     }
 }
 ----
-
-An example return value, assume you create this KEY "my_atomic_counter:1" DOC {"count": 23}
-if the Eventing function above is deployed the count will be immediately incremented :
-
+.Results
 [source,javascript]
 ----
 {
     "doc": {
-        "count": 24
+        "count": 22
     },
     "meta": {
         "id": "key::1",
-        "cas": "1609374571840471040"
+        "cas": "1609374770297176064"
     },
     "success": true
 }
 ----
+====
 
-* _binding_
-+
-The name of the binding that references the target bucket. The binding must have access level of _"read/write"_.
+[#advanced-touch-op]
+== Advanced TOUCH Operation
```

Comment:
Hi @jon-strabala, the only new content on this page is this section about the TOUCH operation. I got the code example operation and response directly from Abhishek (he signed off on the code example + the TOUCH description over slack).

Everything else on the page is the old content -- I just reworked the language and structure. The return values have all been moved out of the individual operation sections and into a table at the bottom of the page.